packages/endo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/endo",
-  "version": "0.0.0+1-dev",
+  "version": "0.1.0",
```

Comment:
Should mark this package as private and rename to `@endo/endo`. It’s not ready for limelight.